default/scripting/species/common/industry.macros

```diff
@@ -5,15 +5,25 @@ BASIC_INDUSTRY
 '''        EffectsGroup
             scope = Source
             activation = And [
-                Source
                 Planet
                 TargetIndustry low = 0
                 Focus type = "FOCUS_INDUSTRY"
             ]
             accountinglabel = "FOCUS_INDUSTRY_LABEL"
-            priority = [[EARLY_PRIORITY]]
+            priority = [[DEFAULT_PRIORITY]]
```

Comment:
No. Default priority is explicitly reserved for the species trait multipliers, due to the vagaries of timing anything that is set to happen at the same time can happen before or after, and getting the balance right on these was a pain. Please keep as Early.